sagemaker-debugger/tensorflow_action_on_rule/detect_stalled_training_job_and_stop.ipynb

```diff
@@ -4,30 +4,22 @@
    "cell_type": "markdown",
```

Comment:
<p>Where can user see rule job logs? </p><p> </p><p>FAQ: Why is my training not stopping? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>This new script outputs the rule job status every 15 minutes as the following:</p><pre class="ql-syntax" spellcheck="false">Current job status: [PrimaryStatus: Stopped, SecondaryStatus: Stopped] | StalledTrainingRule Rule Evaluation Status: IssuesFound
</pre><p>I created this output script to match the descriptions, and customers can easily read it.</p><p> </p><p> </p><p>The previous script for CW, <code>get_rule_jobs_cw_urls(estimator)</code>, outputs:</p><pre class="ql-syntax" spellcheck="false">{'StalledTrainingRule': 'https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#logStream:group=/aws/sagemaker/ProcessingJobs;prefix=smdebug-stalled-demo-16019-StalledTrainingRule-f8bac2de;streamFilter=typeLogStreamPrefix'}
</pre><p>It is unclear what to do with this url. I can add a statement to check CW if they want to look far a detailed rule job logs.</p>

Reply:
We should provide detailed steps on how to view CW logs for a rule.  Actually it should be in sagemaker doc page doc.

Reply:
<p>I recently added this page, and provided instruction to check the rule job logs: https://docs.aws.amazon.com/sagemaker/latest/dg/debugger-cloudwatch-lambda.html</p>

Reply:
This doc doesn't tell how to get to actual CW log. We should write exact steps in the docs.
For example, customer go to training job, he sees 1000s of training job. 
He goes to processing job he sees 1000s of processing job. How should he go ahead and find the processing job that he is interested in. 
How does he get to logs from ProcessingJob page?

Reply:
I see, good point. I will include it to the doc and the notebook again.

Reply:
Incorporated